server/index.js

```diff
@@ -20,7 +20,10 @@ const { AppType } = require('../shared/app');
 const bluebird = require('bluebird');
 const savePdf = require('./lib/pdf-export');
 
+const serverMonitorAnimation = require('../examples/animations/server-monitor').create();
```

Comment:
I don't think that the code specific to the examples should be automatically ran. I would suggest adding a field to the configuration yaml file which would enable this example (it would be disabled in the default configuration).